engine-rest/engine-rest-openapi/src/main/templates/paths/history/process-instance/{id}/get.ftl

```diff
@@ -46,7 +46,8 @@
                                   "superCaseInstanceId":null,
                                   "caseInstanceId":null,
                                   "tenantId":null,
-                                  "state":"ACTIVE"
+                                  "state":"ACTIVE",
+                                  "restartedProcessInstanceId":"2bef365d-3406-11ef-bd73-0a0027000003"
```

Comment:
Validated in swagger editor
<details>
<summary>Screenshot</summary>
<img width="1268" alt="Screenshot 2024-09-09 at 4 27 26 PM" src="https://github.com/user-attachments/assets/fdd84396-911d-4bdd-8013-105ef44bbfd2">

</details>